Tactical/Rotting Corpses.cpp

```diff
@@ -1544,52 +1540,6 @@ ROTTING_CORPSE  *FindCorpseBasedOnStructure( INT32 sGridNo, INT8 asLevel, STRUCT
 
 void CorpseHit( INT32 sGridNo, INT8 asLevel, UINT16 usStructureID )
 {
-#if 0

```

Comment:
Preserve this and let's try to enable twitching corpses when shooting at them? :)

Reply:
I'll create an issue for it